### PR TITLE
Andromeda Unification

### PR DESCRIPTION
**Pull request by kaleb-keny**

### CODE REVIEW

File: tomls/omnibus-base-mainnet-andromeda/collaterals/susdc.toml [hunk 1]
```diff
@@ -5,7 +5,7 @@ defaultValue = "<%= MaxUint256 %>"
 defaultValue = "<%= parseEther('1.005') %>"
```
Comment (cavalier-eth) [inline tomls/omnibus-base-mainnet-andromeda/collaterals/susdc.toml:5]: @sunnyvempati if im looking at the right variable, SIP-348 has USDC liqudiation at 110% https://sips.synthetix.io/sips/sip-348/

File: omnibus-base-mainnet-andromeda.toml [hunk 2]
```diff
@@ -15,7 +15,7 @@ include = [
     "tomls/omnibus-base-mainnet-andromeda/perps/referrers.toml",
     "tomls/omnibus-base-mainnet-andromeda/perps/global.toml",
     "tomls/oracles/perps-keeper-cost.toml", # Add gas oracle for keeper fees
-#    "tomls/omnibus-base-goerli-andromeda/perps/feeCollector.toml", ?? does not exist on mainnet
+#    "tomls/omnibus-base-mainnet-andromeda/perps/feeCollector.toml", ?? commented about until SIP-345 released
```
Comment (cavalier-eth) [inline omnibus-base-mainnet-andromeda.toml:18]: feeCollector.toml exists on mainnet but im unsure if its being released until https://github.com/Synthetixio/synthetix-v3/pull/1969 is finalized

File: tomls/omnibus-base-mainnet-andromeda/perps/feeCollector.toml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 [setting.snx_address]
-defaultValue = ""
+defaultValue = "0x22e6966B799c4D5B13BE962E1D117b56327FDa66"
```
Comment (cavalier-eth) [inline tomls/omnibus-base-mainnet-andromeda/perps/feeCollector.toml:2]: as deployed by https://sips.synthetix.io/sips/sip-336/ 0x22e6966B799c4D5B13BE962E1D117b56327FDa66

File: omnibus-base-mainnet-andromeda.toml [hunk 4]
```diff
@@ -1,38 +1,33 @@
...
-    # Oracles
-    "tomls/oracles/pyth-btc.toml",
     "tomls/oracles/pyth-eth.toml",
-    "tomls/oracles/perps-keeper-cost.toml", # Add gas oracle for keeper fees
-    # "tomls/oracles/pyth-snx.toml",
-    # Permissions
-    "tomls/permit-all-perps-perpsSystem.toml", # Enable perps market
-
+    "tomls/omnibus-base-mainnet-andromeda/perps/eth.toml",
+#    # SNX ?? does not exist on mainnet
```
Comment (cavalier-eth) [inline omnibus-base-mainnet-andromeda.toml:29]: @kaleb-keny i might be misundertanding by the pyth oracle contract on base seems to be https://basescan.org/address/0x8250f4aF4B972684F7b336503E2D6dFeDeB1487a and the toml seems to exist

File: tomls/omnibus-base-mainnet-andromeda/collaterals/susdc.toml (hunk 1 above)
Comment (kaleb-keny) [reply tomls/omnibus-base-mainnet-andromeda/collaterals/susdc.toml:5]: yeah it shouldn't be this i will update the sip today, with borrow disabled it doesn't need to be high

File: omnibus-base-mainnet-andromeda.toml (hunk 2 above)
Comment (kaleb-keny) [reply omnibus-base-mainnet-andromeda.toml:18]: hey @cavalier-eth yeah won't be released initially but i kept this part as a placeholder and in order to be make the goerli / mainnet toml files as similar to each other as possible

File: omnibus-base-mainnet-andromeda.toml (hunk 4 above)
Comment (kaleb-keny) [reply omnibus-base-mainnet-andromeda.toml:29]: This was not included in the mainnet release, but was included in the goerli release

**Pull request merged**